### PR TITLE
[FIX] Image lazy load was breaking attachments

### LINKED ISSUE #10895: The new Lazy load image attachments doesn't work
### Description:

According to #10608, our instance of RC doesn't show the image when we point at it. Is there a way to disable this Lazy load feature?

### Server Setup Information:

* Version of Rocket.Chat Server: 0.65.0
* Operating System: Ubuntu 16.04.1 LTS
* Deployment Method(snap/docker/tar/etc): docker (official rocketchat/rocket.chat)
* Number of Running Instances: 1
* DB Replicaset Oplog: Disabled
* Node Version: n/a
* mongoDB Version: n/a

### Steps to Reproduce:

1. Upload image

### Expected behavior:

Point at the preview image to see the real image

### Actual behavior:

Nothing happened when pointing the preview image
screen shot 2561-05-28 at 10 25 55

### Relevant logs:
n/a

### PR DESCRIPTION
**Pull request by ggazzo**
List to check

- [x] Avatars

- [x] Attachments 

Closes #10895

### CODE REVIEW

File: packages/rocketchat-lazy-load/client/index.js [hunk 1]
```diff
@@ -24,23 +24,29 @@ export const fixCordova = function(url) {
...
 	img.onload = function() {
-		instance.loaded.set(true);
-		instance.firstNode.removeAttribute('data-src');
+		if (instance) {
+			instance.loaded.set(true);
+		} else {
+			element.className = element.className.replace('lazy-img', '');
+			element.src = src;
+		}
+		el.removeAttribute('data-src');
```
Comment (sampaiodiego) [inline packages/rocketchat-lazy-load/client/index.js]: change `el` to `element`

**Comment by JSzaszvari** [comment]
@ggazzo Not sure if this is related, but seeing this as well when running tests as well as my staging environment.

When searching for a user, a good majority of the time none of the avatars show up like this:

image



Happy to log a separate bug report if you'd like, But not sure if it's related to this issue.

**Comment by ggazzo** [comment]
thanks @JSzaszvari I think its working now :x

**Comment by ggazzo** [comment]
<3

**Comment by tsukiRep** [comment]
When can we expect this to be merged?

**Comment by JSzaszvari** [comment]
Amazing @ggazzo - Working great now

**Comment by vynmera** [comment]
Please add this as hotfix, it is currently causing a lot of issues on my team (we have to resort to the insecure imgur now, or tell people to download the images)
Thank you!

**Comment by glebtv** [comment]
This is a user script to work around this issue:

```
setInterval(function() {
	document.querySelectorAll(".attachment-image img").forEach(function(v) { if (v.src != v.dataset.src) {v.src = v.dataset.src; } })
}, 1000)
```

**Pull request merged**